commit 6c06f6a6b48d7f1756860120d21d5ede72735bf6
Author: Thorsten Blum <thorsten.blum@linux.dev>
Date:   Mon Oct 7 13:49:36 2024 +0200

    MAINTAINERS: Add kernel hardening keywords __counted_by{_le|_be}
    
    In addition to __counted_by, also match the keywords __counted_by_le and
    __counted_by_be.
    
    Signed-off-by: Thorsten Blum <thorsten.blum@linux.dev>
    Link: https://lore.kernel.org/r/20241007114935.1647-2-thorsten.blum@linux.dev
    Signed-off-by: Kees Cook <kees@kernel.org>

diff --git a/MAINTAINERS b/MAINTAINERS
index a097afd76ded..cfb32cc22673 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -12349,7 +12349,7 @@ F:	mm/usercopy.c
 F:	security/Kconfig.hardening
 K:	\b(add|choose)_random_kstack_offset\b
 K:	\b__check_(object_size|heap_object)\b
-K:	\b__counted_by\b
+K:	\b__counted_by(_le|_be)?\b
 
 KERNEL JANITORS
 L:	kernel-janitors@vger.kernel.org